Makefile

```diff
@@ -95,3 +101,9 @@ install-openscap:
 .PHONY: vagrant-check
 vagrant-check:
 	BOX=$(BOX) sh ./vagrant.sh 
+
+.PHONY: build-skopeo
+build-skopeo:
+	rm -rf $(DESTDIR)/skopeo
+	git clone --depth 1 $(SKOPEO_SRC_REPO) -b $(SKOPEO_SRC_BRANCH) $(DESTDIR)/skopeo
+	make binary --directory $(DESTDIR)/skopeo -d
```

Comment:
This was stub'd out when we had dependencies on unreleased skopeo features. I'm not calling this in make test but left it in for local dev convenience. Not sure if it's useful or not.